flow-libs/pirates.js.flow

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+// Derived from : https://github.com/ariporad/pirates/blob/master/index.d.ts
+// Which is licensed MIT.
+type Options = {|
+  matcher?: Matcher,
+  exts?: Array<string>,
+  ignoreNodeModules?: boolean,
+|};
```

Comment:
Move this into the `declare module` block. Otherwise it's a global type

Reply:
ok updated this 🙇  @mischnic